plenum/server/primary_selector.py

```diff
@@ -68,24 +68,24 @@ def next_primary_replica_name_for_master(self, nodeReg=None, node_ids=None):
         But since the view number is incremented by 1 before primary selection
         then current approach may be treated as round robin.
         """
-        name = self._next_primary_node_name_for_master(nodeReg, node_ids)
+        name = self._next_primary_node_name_for_master(node_reg, node_ids)
         return name, Replica.generateName(nodeName=name, instId=0)
 
     def next_primary_replica_name_for_backup(self, instance_id, master_primary_rank,
```

Comment:
Why did we remove default values for `node_reg` and `node_ids` here? Currently the callers of this method had to pass some internals (like `self._node.nodeReg` and `self._node.poolManager._ordered_node_ids`)

Reply:
Probably we will not call this with node's nodeReg and odrdered_node_ids. It will be called only in future primaries. When committed we will take primaries from audit ledger